invenio_app_ils/vocabularies/data/alternative_title_types.json

```diff
@@ -1,22 +1,22 @@
 [
   {
     "type": "alternative_title_type",
-    "key": "AlternativeTitle",
+    "key": "ALTERNATIVE_TITLE",
```

Comment:
@equadon @kprzerwa what do you thing to use the uppercase instead?

Reply:
We use uppercase elsewhere for constants in ILS so we should continue doing that I think.